angular/app/pages/landing/landing.scss

```diff
@@ -98,7 +96,7 @@ body{
 .DemoCode{
     font-family: "Lucida Console", Monaco, monospace;
     overflow: auto !important;
-    padding: 10px;
+    padding: 10px 10px 10px 20px;
```

Comment:
it's minor but I'm not sure if it affects the design properly
If it makes it better, keep it 😄